Examples/HttpAnalyzer/HttpStatsCollector.h

```diff
@@ -135,9 +134,10 @@ class HttpStatsCollector
 	/**
 	 * C'tor - clear all structures
 	 */
-	HttpStatsCollector()
+	HttpStatsCollector(unsigned short dstPort)

```

Comment:
instead of `unsigned short` you can use `uint16_t` which is the convention in the rest of the library